nifi-assembly/pom.xml

```diff
@@ -19,6 +19,10 @@ language governing permissions and limitations under the License. -->
     <artifactId>nifi-assembly</artifactId>
     <packaging>pom</packaging>
     <description>This is the assembly Apache NiFi</description>
+    <!--Add a blank run.as in case RPM isn't being used -->
+    <properties>
+        <nifi.run.as />
```

Comment:
@trixpan So from my point of view running the rpm profile should be additive and not change the .tar.gz or .zip. I think we can use installScriptlet or postinstallScriptlet to do a sed replace of the run.as= and not have to fuss with modifying bootstrap.conf outside of the rpm. Thoughts?